01-wasmcloud_basics/wash_intro/claims.md

```diff
@@ -0,0 +1,47 @@
+## wash claims
+
+The `wash claims` subcommand is used for generating and managing JWTs for wasmcloud actors.
+
+### wash claims sign
+
+The `wash claims sign` command allows the user to sign a wasm file and couple it with `wasmcloud` capabilities.
```

Comment:
mentioned elsewhere - don't use backticks around wasmcloud unless we're talking about the binary.